js-envs-test-kit/src/main/scala/org/scalajs/jsenv/test/TimeoutComTests.scala

```diff
@@ -17,9 +17,16 @@ private[test] class TimeoutComTests(config: JSEnvSuiteConfig) {
     assumeTrue("JSEnv needs com support", config.supportsCom)
   }
 
+  /** Slack for timeout tests (see #3457)
+   *
+   *  Empirically we can observe that timing can be off by ~0.1ms. By cutting
+   *  10ms slack, we definitely account for this without compromising the tests.
+   */
+  private val slack = 10.millis
```

Comment:
I wonder if we should make this part of `JSEnvSuiteConfig`?

Reply:
I don't think we should move it to `JSEnvSuiteConfig`, no. At least not yet. If we discover a case later, we can always move it to the Config later in a binary compatible way, so I'd refrain from over-generalizing too early.